test/c3/appframework/appframework_test.go

```diff
@@ -424,4 +464,171 @@ var _ = Describe("c3appfw test", func() {
 			testenv.VerifyAppInstalled(deployment, testenvInstance, testenvInstance.GetName(), podNames, appListV1, false, "enabled", false, true)
 		})
 	})
+
```

Comment:
Does this contain scenario to downgrade apps ?

Reply:
No just upgrade but good catch. Let me do that

Reply:
Added